frontend/src/index.css

```
@@ -1,70 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Space+Mono:wght@400;700&display=swap');
```

Stylelint (17.3.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Space+Mono:wght@400;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Space+Mono:wght@400;700&display=swap'" (import-notation)

(import-notation)